config/settings.yml

```diff
@@ -27,6 +27,11 @@ github_notification_monitor:
   repo_names: []
 merge_target_titler:
   enabled_repos: []
+path_based_labeler:
+  enabled_repos: []
+  rules: []
+    # - pattern:
+    #   label:
```

Comment:
I was just about to comment on this given the other PR ^_^

So I started that way but ran in to the `enabled_repos` behavior, wherever that is coming from. I wasn't sure how to structure the yml in a way that made sense. What do you expect the yml to look like with these rules and `enabled_repos`?